### PR TITLE
Update index.d.ts

### PR DESCRIPTION
**Pull request by mfix-stripe**
Closes https://github.com/markdoc/markdoc/discussions/454

### CODE REVIEW

**Comment by mfix-stripe** [comment]
Hey @stefanprobst, does this work for your use case?

File: src/index.d.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
+import {NextConfig} from 'next';
```
Comment (stefanprobst) [inline src/index.d.ts:1]:
```suggestion
import type {NextConfig} from 'next';
```

File: src/index.d.ts [hunk 2]
```diff
@@ -17,3 +19,19 @@ export type MarkdocNextJsSchema<O extends Object = {}> = Schema<
...
+    allowComments?: boolean;
+  };
+  schemaPath?: string;
+}
+
+declare function createMarkdocPlugin(
+  options?: MarkdocNextJsOptions
+): (config: NextConfig) => NextConfig;
+
+export = createMarkdocPlugin;
```
Comment (stefanprobst) [inline src/index.d.ts:37]: hmm ts says: "TS2309: An export assignment cannot be used in a module with other exported elements."

Comment (stefanprobst) [reply src/index.d.ts:37]: but it does seem to work.

Comment (mfix-stripe) [reply src/index.d.ts:37]: I'll try `export default` instead

**Comment by mfix-stripe** [comment]
Thanks for the review @stefanprobst! I really appreciate it. Made the updates you suggested—lmk what you think.

**Review by stefanprobst — APPROVED** [review]
LGTM :+1:

checked in a vanilla create-next-app and in my own project.

**Comment by stefanprobst** [comment]
@mfix-stripe when do you plan to release this?

**Comment by mfix-stripe** [comment]
Thanks for the nudge @stefanprobst — releasing right now.

**Pull request merged**